modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/AbstractSnapshotSelfTest.java

```diff
@@ -130,7 +131,7 @@
                 .setPageSize(4096))
             .setCacheConfiguration(dfltCacheCfg)
             .setClusterStateOnStart(INACTIVE)
-            .setIncludeEventTypes(EVTS_CLUSTER_SNAPSHOT)
+            .setIncludeEventTypes(EVTS_ALL)
```

Comment:
Do we really need all events? Looks like there no new events needed in new tests.

Reply:
Fixed.